### PR TITLE
added a new category 11

### PR DESCRIPTION
**Pull request by Parikshit-Hooda**
added great youtube channels to follow

### CODE REVIEW

**Comment by Contrevien** [comment]
Looks great to me now! Check the channels once and merge at will

File: README.md [hunk 1]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
-# 11. Additional Links
+      
+     
+============================================================================================
+
+# 11. Recommended YouTube Channels to Follow
+
+
+- [7-second riddles](https://www.youtube.com/channel/UCttFk8-Nysnyw59aNlWOWzw)
+- [99U](https://www.youtube.com/channel/UCVpIzLeRuPW5xy4iBum3IPw)
```
Comment (Contrevien) [inline README.md]: Please review this channel (99U)

File: README.md [hunk 2]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [Abdul Bari](https://www.youtube.com/channel/UCZCFT11CWBi3MHNlGf019nw)
+- [Algorithms with Rachit Jain](https://www.youtube.com/channel/UC9fDC_eBh9e_bogw87DbGKQ)
+- [Angular Firebase](https://www.youtube.com/channel/UCsBjURrPoezykLs9EqgamOA)
+- [Be Smarter Faster](https://www.youtube.com/channel/UCfzLlzRGxSL487ZsK1Q9M4Q)
```
Comment (Contrevien) [inline README.md]: Please review this channel (Be Smarter Faster)

File: README.md [hunk 3]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [Beyond Science](https://www.youtube.com/channel/UCxo8ooAqXiObjuaIy10ud0A)
```
Comment (Contrevien) [inline README.md]: Not qualified as a coding resource

File: README.md [hunk 4]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [Big Think](https://www.youtube.com/channel/UCvQECJukTDE2i6aCoMnS-Vg)
```
Comment (Contrevien) [inline README.md]: Please review (Big Think)

File: README.md [hunk 5]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [BlitheringGenius](https://www.youtube.com/channel/UCbDBo1nfPbSa_UJ_9z0Q3sA)
```
Comment (Contrevien) [inline README.md]: Not qualified as a coding resource

File: README.md [hunk 6]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [Bright Side](https://www.youtube.com/channel/UC4rlAVgAK0SGk-yTfe48Qpw)
```
Comment (Contrevien) [inline README.md]: Not qualified as coding resource

File: README.md [hunk 7]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [Byte By Byte](https://www.youtube.com/channel/UCWSYAntBbdd2SLYUqPIxo0w)
```
Comment (Contrevien) [inline README.md]: Please review (Byte By Byte)

File: README.md [hunk 8]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [Cave of Programming](https://www.youtube.com/channel/UCnAdXkr17iQS8YcYl0LhPdw)
+- [Codechef](https://www.youtube.com/channel/UCmk2YHXZQk_3GsLKBqsZoBQ)
+- [Codecourse](https://www.youtube.com/channel/UCpOIUW62tnJTtpWFABxWZ8g)
+- [Coder's Guide](https://www.youtube.com/channel/UCmZ61pTvEj7a2-Z8rFzovZQ)
```
Comment (Contrevien) [inline README.md]: Can't add this, thenewboston would be a better option than this

File: README.md [hunk 9]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [Coding Tech](https://www.youtube.com/channel/UCtxCXg-UvSnTKPOzLH4wJaQ)
+- [Computerphile](https://www.youtube.com/channel/UC9-y-6csu5WGm29I7JiwpnA)
+- [CppCast](https://www.youtube.com/channel/UCuCjADS4u3uJDTqUaG0H9dA)
```
Comment (Contrevien) [inline README.md]: Please review (CppCast)

File: README.md [hunk 10]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [CS Dojo](https://www.youtube.com/channel/UCxX9wt5FWQUAAz4UrysqK9A)
+- [CS50](https://www.youtube.com/channel/UCcabW7890RKJzL968QWEykA)
+- [CSBreakdown](https://www.youtube.com/channel/UCyJfwPzs4rDuxFd_Ix5cxBw)
```
Comment (Contrevien) [inline README.md]: Review (CSBreakdown)

File: README.md [hunk 11]
```diff
@@ -439,9 +440,47 @@ When I was in College, I missed a lot of opportunties like hackathons, conferenc
...
+- [Derek Banas](https://www.youtube.com/channel/UCwRXb5dUK4cvsHbx-rGzSgw)
+- [Designcourse](https://www.youtube.com/channel/UCVyRiMvfUNMA1UPlDPzG5Ow)
+- [DevTips](https://www.youtube.com/channel/UCyIe-61Y8C4_o-zZCtO4ETQ)
+- [Entrepreneur](https://www.youtube.com/channel/UCNFGSWVOdVWEe9XJNnfTdyQ)
```
Comment (Contrevien) [inline README.md]: Not a qualified coding resource

**Comment by dipakkr** [comment]
Thanks for review @Contrevien. Closing this PR. It won't add much value to the repository.

**Pull request closed without merging**